src/components/banner/banner.scss

```diff
@@ -1,18 +1,79 @@
-@use '../../app/globals';
+@use '../../app/globals' as *;
 
-body {
-  .banner {
-    text-align: center;
-    background-color: globals.$tertiary-color;
-    height: 7rem;
+.banner-container {
+  min-height: 75vh;
```

Comment:
```suggestion
```

This will more closely match the Figma design:

## Design

![image](https://github.com/OpenBCca/openbc-web/assets/16601729/0d898bf7-33f6-45f9-8893-a3ff664b8b67)

## With `min-height`

![image](https://github.com/OpenBCca/openbc-web/assets/16601729/a1cb9410-daa6-4573-ba31-b41757ba1858)

## Removal of min-height

![image](https://github.com/OpenBCca/openbc-web/assets/16601729/8de6fb9f-65ba-4e0c-a5c2-dea5ca854846)

Reply:
Addressed in [dc3ac4e](https://github.com/OpenBCca/openbc-web/pull/92/commits/dc3ac4ea3e52efd14a062d89377a9fed4550450c)